src/Control/Monad/Reader.purs

```diff
@@ -5,13 +5,13 @@ module Control.Monad.Reader
   , runReader
   , mapReader
   , withReader
-  , module Control.Monad.Reader.Class
+  , module X
```

Comment:
Without exporting as an alias, Control.Monad.Reader.Class is marked as redundant. Is that desired behavior?